math/libfixmath/Portfile

```
@@ -0,0 +1,28 @@
+# -*- coding: utf-8; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+PortGroup           github 1.0
+
+github.setup        PetteriAimonen libfixmath ddff910fc03c6e9581ed186268b508dc0f51f4b7
+version             2022.12.27
+revision            0
+categories          math
+license             MIT
+maintainers         {@barracuda156 gmail.com:vital.had} openmaintainer
+description         Q16.16 format fixed point operations in C
+long_description    ${description}
```

When reusing `${description}` in `long_description`, you need to use the Tcl expansion operator (`{*}`):

```suggestion
long_description    {*}${description}
```

---

@herbygillot Sure, gimme a min.